Components/toast/toast.html

```diff
@@ -0,0 +1,127 @@
+                </a>
+                <a>
+                    <i id="toggle-theme" class="fas fa-moon icon"></i>
+                </a>
+            </div>
+        </nav>
+        <div id="sidebar-menu" class="side-bar position-hidden">
+            <div class="nested-list stick">
+                <div>
+                    <h1 class="list-topic display-flex">Get Started <span class="margin-left"><i id="close-sidebar"
+                                class="fas fa-times-circle"></i></span></h1>
+                    <ul class="list-subtopic">
+                        <li>
+                            <a class="link-hover" href="../../Documentation/Introduction.html">Introduction</a>
+                        </li>
+                        <li>
+                            <a class="link-hover" href="../../Documentation/Color.html">Colors</a>
+                        </li>
+                        <li>
+                            <a class="link-hover" href="../../Documentation/Typography.html">Typography</a>
+                        </li>
+                        <li>
+                            <a class="link-hover" href="../../Documentation/Conclusion.html">Conclusion</a>
+                        </li>
+                    </ul>
+                </div>
+                <div>
+                    <h1 class="list-topic">Components</h1>
+                    <ul class="list-subtopic">
+                        <li>
+                            <a class="link-hover" href="../Avatar/Avatar.html">Avatar</a>
+                        </li>
+                        <li>
+                            <a class="link-hover" href="../Badge/Badge.html">Badge</a>
+                        </li>
+                        <li>
+                            <a class="link-hover" href="../Alert/Alert.html">Alert</a>
+                        </li>
+                        <li>
+                            <a class="link-hover" href="../Button/Button.html">Button</a>
+                        </li>
+                        <li><a class="link-hover" href="../Card/card.html">Card</a></li>
+                        <li><a class="link-hover" href="../Image/image.html">Image</a></li>
+                        <li><a class="link-hover" href="../Input/input.html">Input</a></li>
+                        <li><a class="link-hover" href="./text-utilites.html">Text Utilities</a></li>
+                        <li><a class="link-hover" href="../lists/lists.html">Lists</a></li>
+                        <li><a class="link-hover" href="../navigation/navigation.html">Navigation</a></li>
+                        <li><a class="link-hover" href="../modal/modal.html">Modal</a></li>
+                        <li><a class="link-hover" href="../rating/rating.html">Rating</a></li>
+                        <li><a class="link-active" href="./toast.html">Toast</a></li>
+                        <li><a class="link-hover" href="../grid/grid.html">Grid System</a></li>
```

Comment:
Maintain consistency here, some list items are in one line & some are not.

Reply:
thats a bug of prettier. I have tried to fix it but when i save it changes back to this layout cant really do much but thanks for pointing that out!